files/image_config/monit/memory_checker

```diff
@@ -59,6 +69,81 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def get_container_id(container_name):
+    """Gets full container ID of the specified container
+    Args:
+        container_name: A string indicates the name of specified container.
+    Returns:
+        container_id: A string indicates the full ID of specified container.
+    """
+    container_id = ""
+
+    get_container_info_cmd = "docker ps --no-trunc --filter name={}".format(container_name)
+
+    command_stdout = get_command_result(get_container_info_cmd)
+
+    for line in command_stdout.splitlines():
+        if container_name in line:
+            container_id = line.split()[0].strip()
+            break
+
+    if not container_id:
```

Comment:
>container_id [](http://example.com/codeflow?start=11&length=12)

The container id should be guid format, can we do assertion, only concern is what if command output returns something else it may not be empty.

Reply:
Made some check and it seems container-id is not GUID actually, which is just unique id created as 
https://github.com/docker-archive/docker-ce/blob/44a430f4c43e61c95d4e9e9fd6a0573fa113a119/components/engine/daemon/names.go#L41